frontend/src/metabase/query_builder/components/view/View.jsx

```diff
@@ -141,6 +141,7 @@ const ViewMainContainer = props => {
           noHeader
           className={CS.spread}
           mode={queryMode}
+          onUpdateQuestion={updateQuestion}
```

Comment:
We need to pass `onUpdateQuestion` to `VisualizationResult` underneath. Luckily, we already have an implementation of `updateQuestion` in the QB.